### ISSUE TITLE
github pypi version mismatch.

### ISSUE THREAD

**Issue by themobiusproject** [pycontribs/python-vagrant#161]
The latest version of this package on pypi is 1.0.0 where it is 1.1.0 here on github. Who would be the correct one to resolve this?

**Comment by konstruktoid** [comment]
Yeah, the pypi push broke... https://github.com/pycontribs/python-vagrant/actions/runs/3649587808
@ssbarnea can you force push this?

**Comment by Paiusco** [comment]
Do we know if this is being considered yet?

**Comment by daeho-ro** [comment]
1.1.0 is required, now setuptools remove `pkg_resources` in version 82 and it breaks build of `python-vagrant` 1.0.0.
